src/main/resources/youtube/settings/host/values/strings.xml

```diff
@@ -63,6 +63,9 @@ Tap here to learn more about DeArrow."</string>
     <string name="revanced_bypass_image_region_restrictions_summary_off">Using original image host.\n\nEnabling this can fix missing images that are blocked in some regions.</string>
     <string name="revanced_bypass_image_region_restrictions_summary_on">Using image host yt4.ggpht.com.</string>
     <string name="revanced_bypass_image_region_restrictions_title">Bypass image region restrictions</string>
+    <string name="revanced_hook_playlist_download_button_title">Override playlist download action</string>
+    <string name="revanced_hook_playlist_download_button_summary_off">Playlist download will behave like the original, and button might not appear.</string>
+    <string name="revanced_hook_playlist_download_button_summary_on">Playlist download button will be present, and the download will be handled by YTDLnis.</string>
```

Comment:
Rename the hook video download button to match with the issues that I have sent u, then move the strings to this positions